sapio/src/contract/abi/object/mod.rs

```diff
@@ -107,43 +114,43 @@ impl ObjectMetadata {
 /// Object holds a contract's complete context required post-compilation
 /// There is no guarantee that Object is properly constructed presently.
 //TODO: Make type immutable and correct by construction...
-#[derive(Serialize, Deserialize, JsonSchema, Clone, Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Object {
     /// a map of template hashes to the corresponding template, that in the
     /// policy are CTV protected
-    #[serde(
-        rename = "template_hash_to_template_map",
-        skip_serializing_if = "BTreeMap::is_empty",
-        default
-    )]
+    // #[serde(
```

Comment:
It's good to keep these so that the names of fields and their variable names in rust stays independent.

the skip_serializing_if also helps keep in compact